float64 easting
float64 northing
int32 context_score
---
float64 easting
float64 northing
bool valid
bool suitable